fbw-a380x/src/systems/systems-host/index.ts

```diff
@@ -101,6 +104,7 @@ class SystemsHost extends BaseInstrument {
     this.backplane.addInstrument('Amu2', this.amu2, true);
     this.backplane.addInstrument('SimAudioManager', this.simAudioManager);
     this.backplane.addInstrument('Xpndr1', this.xpdr1, true);
+    this.backplane.addInstrument('AtsuSystem', this.atsu,  true);
```

Comment:
Why do you want to allow duplicates?

Reply:
No, I don't intend to allow duplicates. Didn't read the doc, I thought it would lead to a better defined behavior.